We have three method papers published, to support our methodology: 

1. [kmer-based, and synteny+orthology+phylogeny-based subgenome phasing](https://doi.org/10.1093/bib/bbad513)
2. [orthologous synteny identification, polyploidy inference, reticulation inference and phylogenomics](https://doi.org/10.1093/nar/gkaf320)
3. [incomplete lineage sorting (ILS) and introgression/hybridization recognizing](https://doi.org/10.1093/hr/uhae330)

Each method paper has presented detailed guideline/pipeline/codes in the above papers and in GitHub repos. 
The following is the examples of pipelines and codes in GitHub:

1. [subgenome phasing](https://github.com/zhangrengang/subgenome_phasing_example) using [WGDI by Dr. Sun, Pengchuan](https://github.com/SunPengChuan/wgdi) and [SubPhaser](https://github.com/zhangrengang/SubPhaser)
2. [orthologous synteny and phylogenomics](https://github.com/zhangrengang/evolution_example/) using [SOI](https://github.com/zhangrengang/soi) and [ASTRAL by Dr. Zhang, Chao](https://github.com/chaoszhang/ASTER)
3. [introgression recognizing](https://github.com/shang-hongyun/phytop_workflow) using [phytop](https://github.com/zhangrengang/phytop)

These tools are user-friendly. Any other requests can be raised in *Issues*.
